paper/tex/introduction.tex

```diff
@@ -1,63 +1,40 @@
-
-%This paper aims to constrain the impact of these differing interim priors and isolating the effects due to the differing code methodologies by using a homogeneous 
-
-
-%This paper therefore aims (1) to constrain the impact of the interim prior $I$ by separating it into a component $I_{H}$ representing the method itself and a component $I_{D}$ representing physical information, such as a training set or SED template library and (2) to present a procedure for evaluating the performance of photo-$z$ codes in generic tests that may include many more systematics in the interim prior $I$.
-%In order to isolate the effects encapsulated by $I_{H}$ of variation between codes from issues with the training set or template library encapsulated by $I_{D}$, we use an identical set of simulated galaxies for every code and construct a template library and training sample that are {\it complete and representative} and shared among all codes; that is, our training sample for machine learning codes is drawn from the same underlying galaxy population as our test set, with no additional selections, and the SED library used for template-based codes is the same as the one used to generate the photometric data.
-%We explore a number of performance metrics in this paper, in the controlled environment of the representative training data in order to establish a method for comparing photo-$z$ PDFs derived by different methods.
-%not to make a conclusion regarding the superiority or even relative favorability of each code but to establish a method for comparing photo-$z$ PDFs derived by different methods.
-%These test conditions set the stage for addressing in a future paper the crucial issue of incomplete and non-representative prior information.
-
-
-
-The outline of the paper is as follows: in \S\,\ref{sec:sims} we present the simulated data set; in \S\,\ref{sec:pzcodes} we describe the current generation codes employed in the paper;  in \S\,\ref{sec:metrics} we discuss the interpretation of photo-$z$ PDFs in terms of metrics of accuracy; in \S\,\ref{sec:results} we show our results and compare the performance of the codes; in \S\,\ref{sec:discussion} we offer our conclusions and discuss future extensions of this work.
+The current and next generations of large-scale galaxy surveys, including the Dark Energy Survey \citep[\proj{DES},][]{Abbott:05}, the Kilo-Degree Survey \citep[\proj{KiDS},][]{de_Jong:13}, Hyper Suprime-Cam Survey \citep[\proj{HSC},][]{Aihara:2018a,Aihara:2018b}, Large Synoptic Survey Telescope \citep[\lsst,][]{Abell:09}, Euclid \citep{Laureijs:11}, and Wide-Field Infrared Survey Telescope \citep[\proj{WFIRST},][]{Green:12}, present a paradigm shift from spectroscopic to photometric galaxy catalogues of substantially larger size at a cost of lacking complete spectroscopically confirmed redshifts ($z$).
```

Comment:
Euclid and WFIRST both include a spectroscopic component, so this should probably be reworded a bit.

Reply:
I took a stab at rewording this but am not sure if it helped.